closure/testing/test/BUILD

```diff
@@ -31,28 +31,31 @@ closure_js_library(
     deps = ["//closure/library"],
 )
 
-closure_js_test(
-    name = "arithmetic_module_test",
-    srcs = ["arithmetic_module_test.js"],
-    defs = ["--entry_point=goog:arithmetic_module_test"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
-
-closure_js_test(
-    name = "arithmetic_scope_test",
-    srcs = ["arithmetic_scope_test.js"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
+# TODO(hochhaus): Enable test when upstream local file loader fix is included
```

Comment:
Even if production is ok, we still want our unit testing to work. Can you tell me more about why this is broken? So we can gauge whether or not it's worth raising a fuss for them to cut another release?